src/app/component/core/header-new/header-new.component.ts

```diff
@@ -53,6 +54,13 @@ export class HeaderNewComponent implements OnInit {
       .subscribe(this.assignData.bind(this));
     this.userRole = this.jwtService.getUserRole();
     this.language = this.languageService.getCurrentLanguage();
```

Comment:
Please create a separate fucntion and use it here.

Reply:
fix